studio/src/app/layout.tsx

```
@@ -0,0 +1,88 @@
+      </head>
+      <body>
+        {children}
+
+        <Footer />
+
+        {scripts?.map((script, i) =>
+          script.inline ? (
+            <script
+              key={script.id}
+              id={script.id}
+              dangerouslySetInnerHTML={{ __html: script.src }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)